services/src/main/java/org/keycloak/protocol/oid4vc/issuance/OID4VCIssuerWellKnownProvider.java

```diff
@@ -115,4 +130,30 @@ public static String getIssuer(KeycloakContext context) {
     public static String getCredentialsEndpoint(KeycloakContext context) {
         return getIssuer(context) + "/protocol/" + OID4VCLoginProtocolFactory.PROTOCOL_ID + "/" + OID4VCIssuerEndpoint.CREDENTIAL_PATH;
     }
+
+    private static final String VC_KEY = "vc";
+
+    public static List<SupportedCredentialConfiguration> fromRealmAttributes(Map<String, String> realmAttributes) {
+
+        Set<String> supportedCredentialIds = new HashSet<>();
+        Map<String, String> attributes = new HashMap<>();
+        realmAttributes
+                .entrySet()
+                .forEach(entry -> {
```

Comment:
This part can be written as follows.
```
        realmAttributes
                .forEach((entryKey, value) -> {
                    if (!entryKey.startsWith(VC_KEY)) {
                        return;
                    }
                    String key = entryKey.substring((VC_KEY + ".").length());
                    supportedCredentialIds.add(key.split("\\.")[0]);
                    attributes.put(key, value);
                });
```
I am not sure whether the code above decreases computation compared with the original code, so it is up to you to apply the code above.